README.md

```diff
@@ -14,7 +14,6 @@ simply run:
 
 ```sh
 npm install
-grunt
```

Comment:
how about defining both `prepublish` and `prepare`? If I read the deprecation note correctly, it would make the package able to work correctly across the npm versions, am I right?